package.json

```diff
@@ -203,9 +203,37 @@
   },
   "devDependencies": {
     "@babel/core": "7.0.0-beta.56",
+    "@babel/generator": "7.0.0-beta.56",
+    "@babel/plugin-proposal-class-properties": "7.0.0-beta.56",
+    "@babel/plugin-proposal-object-rest-spread": "7.0.0-beta.56",
+    "@babel/plugin-proposal-optional-chaining": "7.0.0-beta.56",
+    "@babel/plugin-transform-arrow-functions": "7.0.0-beta.56",
+    "@babel/plugin-transform-block-scoping": "7.0.0-beta.56",
+    "@babel/plugin-transform-classes": "7.0.0-beta.56",
+    "@babel/plugin-transform-computed-properties": "7.0.0-beta.56",
+    "@babel/plugin-transform-destructuring": "7.0.0-beta.56",
+    "@babel/plugin-transform-exponentiation-operator": "7.0.0-beta.56",
+    "@babel/plugin-transform-flow-strip-types": "7.0.0-beta.56",
+    "@babel/plugin-transform-for-of": "7.0.0-beta.56",
+    "@babel/plugin-transform-function-name": "7.0.0-beta.56",
+    "@babel/plugin-transform-literals": "7.0.0-beta.56",
+    "@babel/plugin-transform-modules-commonjs": "7.0.0-beta.56",
+    "@babel/plugin-transform-object-assign": "7.0.0-beta.56",
+    "@babel/plugin-transform-parameters": "7.0.0-beta.56",
+    "@babel/plugin-transform-react-display-name": "7.0.0-beta.56",
+    "@babel/plugin-transform-react-jsx": "7.0.0-beta.56",
+    "@babel/plugin-transform-react-jsx-source": "7.0.0-beta.56",
+    "@babel/plugin-transform-regenerator": "7.0.0-beta.56",
+    "@babel/plugin-transform-shorthand-properties": "7.0.0-beta.56",
+    "@babel/plugin-transform-spread": "7.0.0-beta.56",
+    "@babel/plugin-transform-sticky-regex": "7.0.0-beta.56",
+    "@babel/plugin-transform-template-literals": "7.0.0-beta.56",
+    "@babel/plugin-transform-unicode-regex": "7.0.0-beta.56",
+    "@babel/plugin-transform-runtime": "7.0.0-beta.56",
+    "@babel/runtime": "7.0.0-beta.56",
     "async": "^2.4.0",
     "babel-eslint": "9.0.0-beta.2",
-    "babel-generator": "^6.26.0",
+    "babel-preset-react-native": "^5.0.2",
```

Comment:
Why is this not `"metro-react-native-babel-preset": "0.43.5"`?

Reply:
Ah, I could not find where it had moved to as per my comment in the PR. Will update 👍

Reply:
ah yeah sorry I'm dumb 🙈